signal_scanner_bot/signal.py

```diff
@@ -45,6 +44,45 @@ def message_timestamp(data: Dict) -> datetime:
     return dt
 
 
+def list_identities() -> str:
+    """
+        Function that calls the signal-cli `listIdentities` command and returns the entire result as a string
+    """
+    with env.SIGNAL_LOCK:
+        proc = subprocess.run(
+            [
+                "signal-cli",
+                "-u",
+                str(env.BOT_NUMBER),
+                "listIdentities"
+            ],
+            capture_output=True,
+        )
+    if proc.stdout:
+        return proc.stdout.decode('utf-8')
+    if proc.stderr:
+        log.warning(f"STDERR: {proc.stderr.decode('utf-8')}")
```

Comment:
This might need to come before the `stdout` if block, since that will return if anything is found and these lines would never be reached